CDP4SiteDirectory/ViewModels/OrganizationBrowser/OrganizationBrowserViewModel.cs

```diff
@@ -113,7 +113,7 @@ private void UpdatePersonRows(OrganizationRowViewModel organizationRow)
 
             foreach (var addedPerson in addedPersons)
             {
-                var row = new OrganizationBrowser.PersonRowViewModel(addedPerson, this.Session, this);
+                var row = new OrganizationBrowser.PersonRowViewModel(addedPerson, this.Session, organizationRow);
```

Comment:
I don't understand. Is this correct?

Reply:
Yes, `this` in this case is the `OrganizationBrowserViewModel` which is incorrect vm for the `ContainerRow` of the `PersonRowViewModel`.